Atomic/storage.py

```diff
@@ -19,6 +19,30 @@
 except ImportError:
     from atomic import Atomic
 
+def modify_sh_var_in_text(text, var, modifier, default=""):
+    pattern = '^[ \t]*%s[ \t]*=[ \t]*"(.*)"[ \t]*$' % re.escape(var)
+    found = [ False ]
+    def sub(match):
```

Comment:
There is `re.subn` which gives the number of substitutions, so I have changed the code to use that instead.